docs/upload_docs.py

```diff
@@ -1,90 +0,0 @@
-import argparse
```

Comment:
Since this is deleted, the build_docs.sh will fail.

Reply:
This is now resolved in [PR 1167]( https://github.com/apple/coremltools/pull/1167).